controllers/subscriptionsController/deleteAppSubscription.js

```diff
@@ -0,0 +1,73 @@
+//utilities
+const CustomError = require("../../utils/customError");
+const responseHandler = require("../../utils/responseHandler");
+
+//models
+const OrganizationModel = require("../../models/organizations");
+const ApplicationModel = require("../../models/applications");
+const SubscriptionModel = require("../../models/subscriptions");
+const SubscriptionHistoryModel = require("../../models/subscriptionsHistory");
+
+/**
+ * @author Ekeyekwu Oscar
+ *
+ * deletes an application subscription
+ *
+ * @param {*} req - The request object
+ * @param {*} res - The response object
+ * @param {*} next - The function executed to call the next middleware
+ */
+
+const deleteAppSubscription = async (req, res, next) => {
+  const { applicationId } = req.params;
+  const { organizationId } = req.token;
+
+  try {
+    //check if organization exists
+    const organization = await OrganizationModel.findById(organizationId);
+    if (!organization) {
+      next(new CustomError("404", "Organization not found"));
+      return;
+    }
+
+    //check if application exists
+    const application = await ApplicationModel.findById({
```

Comment:
This should be `find` not `findById`. I'm surprised it didn't break here break here, as findById expects only one param `id`